test/test-npm-install.js

```diff
@@ -68,6 +68,25 @@ test('npm-install: no package.json', function (t) {
   });
 });
 
+test('npm-install: timout', function (t) {
+  var context = {
+    emit: function() {},
+    path: sandbox,
+    module: {
+      name: 'omg-i-pass'
+    },
+    meta: {},
+    options: {
+      npmLevel: 'silly',
+      timeoutLength: 100
+    }
+  };
+  npmInstall(context, function (err) {
+    t.equals(err.message, 'Install Failed');
```

Comment:
Suggest you add this as well; the other tests can perhaps be updated in a sweep at a later stage.